testing/log4cxx/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer:
+# adapted from https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=log4cxx&id=904e7c4f47191e9558cfe26c4d4455a95a0e136d
+
+pkgname=log4cxx
+pkgver=0.10.0
+pkgrel=0
+pkgdesc="A C++ port of the Log4j logging framework."
+url="http://logging.apache.org/log4cxx"
+license="Apache"
+options='!check'  # testsuite in 0.10.0 does not work on alpine
+subpackages="$pkgname-dev"
+depends='apr-util libxml2'
+arch='all'
+makedepends='autoconf automake libtool zip libxml2-dev apr-util-dev'
+source="http://archive.apache.org/dist/logging/$pkgname/$pkgver/apache-$pkgname-$pkgver.tar.gz log4cxx-0.10.0-missing_includes.patch log4cxx-0.10.0-narrowing-fixes-from-upstream.patch"
+builddir="${srcdir}/apache-$pkgname-$pkgver"
+
+build() {
+  cd "${builddir}"
+
+  # if configure is autogenerated, make install seems to fail #./autogen.sh
+  # does not configure if specifying --build=$CBUILD \ --host=$CHOST \
+  ./configure \
+  --prefix=/usr \
+  --sysconfdir=/etc \
+  --mandir=/usr/share/man \
+  --localstatedir=/var
+
+  make
+}
+
+check() {
+  cd "${builddir}"
```

Comment:
This package is in the testing repo, you don't need to have `cd "$builddir"` statements in the prepare() build() check() and package() functions as abuild does that automatically

Reply:
> This package is in the testing repo

Sorry, what does this mean? Is this a new feature of abuild? I was basing this off other APKBUILD files I had found, and examples on the wiki.

Reply:
The package is in the testing directory, when it is built it will be available to people who enabled the testing repository. there are 3 repos `main`, `community` and `testing`.

Yes, implicit `cd "$builddir"` is new.

Reply:
Thanks.